spec/controllers/recordings_controller_spec.rb

```diff
@@ -53,6 +53,46 @@
     end
   end
 
+  describe '#update' do
+    # Metaprogramming -> mocking underlying method implemntation on the run.
+
+    let(:recording_update_params) do
+      {
+        name: 'My Awesome Recording!'
+      }
+    end
+
+    let(:invalid_recording_update_params) do
+      {
+        name: ''
+      }
+    end
```

Comment:
I would remove these and just move the logic into each test case - that way it is clear whats happening